lib/spack/docs/module_file_tutorial.rst

```diff
@@ -774,23 +775,28 @@ After modifications the configuration file will be:
             SLURM_MPI_TYPE: pmi2
             OMPI_MCA_btl_openib_warn_default_gid_prefix: '0'
 
-The double colon after ``enable`` is intentional and it serves the purpose of
-overriding the default list of enabled generators so that only ``lmod`` will be active
-(see :ref:`the reference manual<config-overrides>` for a more detailed explanation
-of config scopes).
+
+.. note::
+  The double colon
+    The double colon after ``enable`` is intentional and it serves the
+    purpose of overriding the default list of enabled generators so
+    that only ``lmod`` will be active (see :ref:`the reference
+    manual<config-overrides>` for a more detailed explanation of
```

Comment:
I think there needs to be a space in between manual and '<' or else the link won't show up.